avocado/plugins/run.py

```diff
@@ -133,7 +133,7 @@ def configure(self, parser):
         loader.add_loader_options(parser)
 
         if multiplexer.MULTIPLEX_CAPABLE:
-            mux = parser.add_argument_group('multiplexer use on test execution')
+            mux = parser.add_argument_group('test parameters')
```

Comment:
This is based on the discussion we had based on the RFC posted to the mailing list, where we decided that, at this point, we'd keep the multiplexer as the only way to process test parameters, right?

The modularity is then left to how a tree can be created and passed to the multiplexer, and after its own processing, to become the test parameters.

Given that my rational (and memory) is correct, I ACK this change.